app/controllers/job_invocations_controller.rb

```diff
@@ -57,7 +57,8 @@ def show
       hosts_base = hosts_base.joins(:template_invocations)
                              .where(:template_invocations => { :job_invocation_id => @job_invocation.id})
     end
-    @hosts = hosts_base.search_for(params[:search], :order => params[:order] || 'name ASC').paginate(:page => params[:page])
+    @hosts = hosts_base.search_for(params[:search], :order => params[:order] || 'name ASC')
```

Comment:
any reason why not use https://github.com/theforeman/foreman/blob/2b0401fc35c5aa0d4e059aba434ade4aafeae2a9/app/controllers/application_controller.rb#L224 ?

Reply:
Mainly that I didn't know we have something like that